trio/_core/_tests/test_asyncgen.py

```diff
@@ -85,10 +92,10 @@ async def async_main():
         assert agen.ag_frame is None  # all should now be exhausted
 
 
-async def test_asyncgen_throws_during_finalization(caplog):
+async def test_asyncgen_throws_during_finalization(caplog) -> None:
```

Comment:
You can annotate the fixture with `pytest.LogCaptureFixture`.

Reply:
Changed in c06e788